### PR TITLE
Move rule handler container into package

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | yiisoft/validator-rule-handler-container#2

### CODE REVIEW

**Comment by xepozz** [comment]
Welcome back, `ContainerRuleHandlerResolver` 🤗

File: src/ContainerRuleHandlerResolver.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+namespace Yiisoft\Validator;
+
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+use Yiisoft\Validator\Exception\RuleHandlerInterfaceNotImplementedException;
+use Yiisoft\Validator\Exception\RuleHandlerNotFoundException;
+use Yiisoft\Validator\Rule\RuleHandlerInterface;
+
+final class ContainerRuleHandlerResolver implements RuleHandlerResolverInterface
```
Comment (vjik) [inline src/ContainerRuleHandlerResolver.php:13]:
```suggestion
final class PsrRuleHandlerContainer implements RuleHandlerResolverInterface
```

File: src/SimpleRuleHandlerResolver.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
 use Yiisoft\Validator\Exception\RuleHandlerInterfaceNotImplementedException;
 use Yiisoft\Validator\Rule\RuleHandlerInterface;
 
-final class SimpleRuleHandlerContainer implements RuleHandlerResolverInterface
+final class SimpleRuleHandlerResolver implements RuleHandlerResolverInterface
```
Comment (vjik) [inline src/SimpleRuleHandlerResolver.php:10]:
```suggestion
final class SimpleRuleHandlerContainer implements RuleHandlerResolverInterface
```

**Review by vjik — COMMENTED** [review]
Class names are just suggestions.

**Review by vjik — APPROVED** [review]

**Pull request merged**